llvm/test/CodeGen/AMDGPU/lds-size-pal-metadata.ll

```diff
@@ -0,0 +1,25 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn--amdpal -mcpu=gfx1200 < %s | FileCheck -check-prefix=PAL %s
+
```

Comment:
Can you comment what this is testing? I'm not sure I understand what the issue is